conda.recipe/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: rise
+  version: 0.3.0
+
+build:
+    number: 1
+
+source:
+  git_url: https://github.com/ijstokes/RISE.git
+  git_tag: master
+
+requirements:
+  build:
+    - python
+    - jupyter
```

Comment:
jupyter is the metackage installing all... there is no need to add it here... only the notebook should be necessary